_RFCs/0003-tracking-package-versions.md

```diff
@@ -0,0 +1,56 @@
+- Change Name: standardize_tracking_major_package_versions
+- Start Date: 2018-03-27
+- RFC PR: (leave this empty)
+- Core-Plans Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+Everytime a new major version of software packaged by Habitat is released (whether stable or an edge release), a new core plan must be made for that version (i.e. when Docker11 is released, we should create core/docker11).
+
+Generic package names (ones which do not specify a version in the name - i.e. core/docker) must always point to the latest **stable** version of the software.  They must not track edge versions of the software.  They must source the plan for the latest stable plan which tracks that software (i.e. if the current stable version of Docker is Docker10, and the edge version is Docker11, core/docker should source core/docker10).
+
+If a user wishes to use an edge version of the software (i.e. Docker11 is the edge version, while Docker10 is the stable version), they can do so by directly depending on core/docker11.
+
+# Motivation
+[motivation]: #motivation
+
+We currently have several generic packages (i.e. core/docker) that track the current stable version of the packaged software (as of today, core/docker points to Docker 17). When a new major version of the package becomes available, we move the old version (i.e. Docker 16) to a separate plan to track the old version (i.e. core/docker16) while the generic package (core/docker) will be updated to point to the current major version of the software.
+
+However, we do not have a process in place to handle edge releases of software. As of today, Docker 18 is the latest edge version of Docker, but the latest stable version is still Docker 17. This brought us to the question - should "core/docker" track the latest stable version or the latest edge version?
+
+# Guide-level explanation
+[guide-level-explanation]: #guide-level-explanation
+
+Many Habitat packages track and package 3rd party software - this includes Docker, Node, Java, and more. We learned early on that even when a new major version of the software was released, many people still used the older version (and sometimes did not have the choice of upgrading immediately due to compliance or other reasons).
+
+Historically, when a new major version of software was released (i.e. when Node 8 became the new LTS over Node 7) we changed the generic package (i.e. core/node) to track the newest major version and, for people who needed to use an older major version of the software, created a new plan (i.e. "core/node7") that they could refer to.
+
+Now we will create a new core plan for every major release of a piece of software, edge or stable (i.e. when Docker11 is released as edge, we will create a core/docker11 plan so that someone can use the edge version if they wish).
```

Comment:
The support burden of  maintaining edge packages seems like an unnecessary burden to me. Primarily I guess I feel like if our users want an edge package they already have everything they need to host an edge package in their own origin which I would think should alleviate a need to increase our overhead.